src/utils/index.js

```diff
@@ -6,7 +6,7 @@ const uint8ArrayEquals = require('uint8arrays/equals')
 /**
  * Creates a logger for the given subsystem
  *
- * @param {import('peer-id')} [id]
+ * @param {any} [id]
```

Comment:
can this be other than PeerID?

Reply:
Oops, left that in by accident - I was toying with the idea of using CIDs instead of PeerIds